### DISCUSSION TITLE
Welcome to Vint Discussions!

### DISCUSSION THREAD

**Discussion by C6OI** [Vint-TX/Vint#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Originickname** [comment]
im lobstr

**Comment by Originickname** [comment]
I'm over here lobstering my vulcan I got lobster on my gun right now I'm just lobstering my crystals I'm lobster as fuck man I'm a lobster man like
